mud-contracts/world/src/modules/inventory/systems/InventoryInteract.sol

```diff
@@ -84,6 +84,64 @@ contract InventoryInteract is EveSystem {
     _inventoryLib().depositToEphemeralInventory(smartObjectId, ephemeralInventoryOwner, inItems);
   }
 
+  /**
+   * @notice Transfer items from ephemeral to inventory
+   * @dev transfer items from ephemeral to inventory
```

Comment:
These comments are incorrect, this is a transfer from inventory to ephemeral

Reply:
fixed